modules/cco-ccoctl-creating-at-once.adoc

```diff
@@ -10,52 +20,100 @@ If you do not need to review the JSON files that the `ccoctl` tool creates befor
+----
++
+where:
++
+** `<name>` is the name used to tag any cloud resources that are created for tracking.
+** `<alibaba-region>` is the Alibaba Cloud region in which cloud resources will be created.
+** `<path_to_directory_with_list_of_credentials_requests>/credrequests` is the directory containing the files for the component `CredentialsRequest` objects.
++
+[NOTE]
+====
+A RAM user can have up to two AccessKeys at the same time. If you run `ccoctl alibabacloud create-ram-users` more than twice, the previous generated manifests secret becomes stale and you must reapply the newly generated secrets.
+====
+endif::alibaba[]
 
 .Verification
 
-* To verify that the {product-title} secrets are created, list the files in the `_<path_to_ccoctl_output_dir>_/manifests` directory:
+* To verify that the {product-title} secrets are created, list the files in the `<path_to_ccoctl_output_dir>/manifests` directory:
 +
 [source,terminal,subs="+quotes"]
 ----
-$ ll __<path_to_ccoctl_output_dir>__/manifests
+$ ls <path_to_ccoctl_output_dir>/manifests
 ----
 +
 .Example output:
 +
 [source,terminal,subs="+quotes"]
+ifdef::aws[]
 ----
-total 24
--rw-------. 1 __<user>__ __<user>__ 161 Apr 13 11:42 cluster-authentication-02-config.yaml
--rw-------. 1 __<user>__ __<user>__ 379 Apr 13 11:59 openshift-cloud-credential-operator-cloud-credential-operator-iam-ro-creds-credentials.yaml
--rw-------. 1 __<user>__ __<user>__ 353 Apr 13 11:59 openshift-cluster-csi-drivers-ebs-cloud-credentials-credentials.yaml
--rw-------. 1 __<user>__ __<user>__ 355 Apr 13 11:59 openshift-image-registry-installer-cloud-credentials-credentials.yaml
--rw-------. 1 __<user>__ __<user>__ 339 Apr 13 11:59 openshift-ingress-operator-cloud-credentials-credentials.yaml
--rw-------. 1 __<user>__ __<user>__ 337 Apr 13 11:59 openshift-machine-api-aws-cloud-credentials-credentials.yaml
+cluster-authentication-02-config.yaml
+openshift-cloud-credential-operator-cloud-credential-operator-iam-ro-creds-credentials.yaml
+openshift-cluster-csi-drivers-ebs-cloud-credentials-credentials.yaml
+openshift-image-registry-installer-cloud-credentials-credentials.yaml
+openshift-ingress-operator-cloud-credentials-credentials.yaml
+openshift-machine-api-aws-cloud-credentials-credentials.yaml
 ----
 
 You can verify that the IAM roles are created by querying AWS. For more information, refer to AWS documentation on listing IAM roles.
+endif::aws[]
+ifdef::alibaba[]
+----
+openshift-cluster-csi-drivers-alibaba-disk-credentials-credentials.yaml
+openshift-image-registry-installer-cloud-credentials-credentials.yaml
+openshift-ingress-operator-cloud-credentials-credentials.yaml
+openshift-machine-api-alibabacloud-credentials-credentials.yaml
+----
+
+You can verify that the RAM users and policies are created by querying Alibaba Cloud. For more information, refer to Alibaba Cloud documentation on listing RAM users and policies.
```

Comment:
For each CredentialsRequest, users should verify that a namespace that matches the text in the spec.secretRef.namespace field exists in the cluster. This is the namespace in which secrets will be created. If it does not exist, users will have to create them, before starting the install process. I remember we adding this to the doc but I do not know where this was added

Reply:
Ack I will look into this!

Reply:
Ok, I see those steps, under 2-3 here: https://github.com/openshift/cloud-credential-operator/blob/master/docs/ccoctl.md#procedure
There is a note that these steps are only applicable to upgrade, not fresh installs. Let me know if that's not right and I can add the step.